docker-compose.yml

```diff
@@ -348,6 +348,21 @@ services:
     deploy:
       mode: global
 
+  # SOCKS5 proxy for use with client
+
+  socks5-proxy:
+    image: serjs/go-socks5-proxy:latest
```

Comment:
can we pin a version here just in case?

Reply:
Done, thanks -> https://hub.docker.com/r/serjs/go-socks5-proxy/tags